app/src/main/java/io/homeassistant/companion/android/widgets/camera/CameraWidget.kt

```diff
@@ -173,15 +173,15 @@ class CameraWidget : AppWidgetProvider() {
     }
 
     private suspend fun retrieveCameraImageUrl(context: Context, entityId: String): String? {
-        val entity = integrationUseCase.getEntity(entityId)
-        if (entity == null) {
+        return try {
+            val entity = integrationUseCase.getEntity(entityId)
+            entity?.attributes?.get("entity_picture")?.toString()
+        } catch (e: Exception) {
             Log.e(TAG, "Failed to fetch entity or entity does not exist")
```

Comment:
now that we have a try/catch block we should make sure to print the exception :)